### PR TITLE
Singer theme

### PR DESCRIPTION
**Pull request by Joren-Thijs**
Custom theme for the singer app.

Only includes colors for now.
known bugs: No styling override for material timepickers in singer event details form.

### CODE REVIEW

File: Singer.API/ClientApp/angular.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
                   "tsConfig": "src/tsconfig.app.json",
                   "assets": ["src/assets"],
                   "styles": [
-                     "src/styles.css"
```
Comment (BerendWouters) [inline Singer.API/ClientApp/angular.json:25]: Since when are we using SASS?

File: Singer.API/ClientApp/src/app/modules/admin/components/singerevents/singerevent-details/singerevent-details.component.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit, Output, EventEmitter, Inject } from '@angular/core';
+import { Component, OnInit, Output, EventEmitter, Inject, NgModule } from '@angular/core';
```
Comment (BerendWouters) [inline Singer.API/ClientApp/src/app/modules/admin/components/singerevents/singerevent-details/singerevent-details.component.ts]: Why is this needed?

**Review by BerendWouters — CHANGES_REQUESTED** [review]
General remark:
- When using SASS (or any other CSS enhancement), please rename all component files accordingly. This means: `admin-list.component.css` -> `admin-list.component.scss`
- I see some styling returning in multiple locations (`th`), please place them in the `styles.scss` file

File: Singer.API/ClientApp/angular.json (hunk 1 above)
Comment (Joren-Thijs) [reply Singer.API/ClientApp/angular.json:25]: Since we need to leverage the angular material theme mixin methods

Comment (Joren-Thijs) [reply Singer.API/ClientApp/angular.json:25]: It's only for our main styles.scss the global project setting still uses css.

Comment (Joren-Thijs) [reply Singer.API/ClientApp/angular.json:25]: If i leave it as a .css file the production build will complain that it does not recognize the $... scss variables in the custom theme.
